contracts/Cryptography.sol

```diff
@@ -0,0 +1,72 @@
+// SPDX-License-Identifier: Apache-2.0
```

Comment:
Where is this file from? Is it disclaimed in the readme?

Reply:
This is our own contract. It was moved here from fuel-v2-contracts. This plus all the tree contracts live in a folder called "lib" there now.